doc/ref_arch/openstack/README.md

```diff
@@ -34,7 +34,7 @@ This is Openstack based Reference Architecture
 
 
 ## Table of Contents
-* [Abbreviations](abbreviations.md)
+* [Terminology](chapter/terminology.md)
```

Comment:
The link doesn't work.

Reply:
Okay I saw that you have added a pending statement related to terminology chapter `Still need to create the chapter and content`. Will it be part of this pull request or a separate one?

Reply:
@deebhatia The Terminology chapter will be created as a separate PR.